manual/index.php

```diff
@@ -85,9 +97,9 @@
 							     	     		     break;
 									};
 
-									echo '<div class="form-row">';

```

Comment:
not sure why, but seems like it was added not in all cases. so move it down to each case.